### ISSUE TITLE
Python: ModuleNotFoundError: No module named 'imp'

### ISSUE THREAD

**Issue by poing** [aws/aws-elastic-beanstalk-cli#493]
The `imp` module has been removed from Python `3.12`.  
https://github.com/python/cpython/issues/98040

This breaks `awsebcli` when using `python:latest` in Docker.

```
$ eb use elastic-beanstalk
Traceback (most recent call last):
  File "/root/.local/bin/eb", line 5, in <module>
    from ebcli.core.ebcore import main
  File "/root/.local/lib/python3.12/site-packages/ebcli/core/ebcore.py", line 16, in <module>
    from cement.core import foundation, handler, hook
  File "/root/.local/lib/python3.12/site-packages/cement/core/foundation.py", line 11, in <module>
    from ..core import output, extension, arg, controller, meta, cache, mail
  File "/root/.local/lib/python3.12/site-packages/cement/core/extension.py", line 8, in <module>
    from imp import reload  # pragma: no cover
    ^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'imp'
```

#### Workaround

Use: `python:3.11`

**Comment by rahulrajaram** [comment]
Thanks for reporting the bug. We're working on a fix for it.

**Comment by rahulrajaram** [comment]
I've fixed the issue upstream in cement: https://github.com/datafolklabs/cement/pull/673. Will release new version of the EB CLI next week. To get around the issue until then install the EB CLI and run: `pip install cement==2.10.14`

**Comment by rahulrajaram** [comment]
Fixed in 3.21. Apologies for the delay.
